addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -434,7 +434,7 @@
                 <group>
```

Comment:
Why do we have a "group" here by the way ? Does it have any utility (aka: can't we simplify the dom), as it seems this is something used by odoo and not really used in standard html ?